src/content/en/android/custom-tabs/headers/index.md

```diff
@@ -0,0 +1,178 @@
+v83 to v85       | whitelisted
+v86 onwards      | whitelisted, non-whitelisted when a digital asset link is set up
+
+**Table 1.:** Filtering of non-whitelisted CORS headers.
+
+
+This guide shows how to set up a verified connection between the server and client and use that to send non-whitelisted http headers. You can skip to [Adding Extra Headers to CustomTab Intents for the code](#adding-extra-headers). 
+
+## Backgroud
+
+### Whitelisted vs. Non-whitelisted CORS Request Headers
+[Cross-Origin Resource Sharing (CORS)][6] allows a web application from one origin to request resources of a different origin. The list of **CORS-whitelisted** headers is maintained in the [HTML Standard][]. Example whitelisted headers are shown in [Table 2](#table-2). 
+
+<a name="table-2"></a>
+
+**Header**       | **Description**
+---------------- | ----------------------
+accept-language  | advertises natural languages the client understands
+content-language | describes language intended for the current audience
+content-type     | indicates the media type of the resource
+
+**Table 2.:** Example whitelisted CORS headers.
+
+The whitelisted headers are considered safe because they don’t contain sensitive user information and are unlikely to cause the server to perform potentially damaging operations.
+
+Examples of non-whitelisted headers are shown in [Table 3](#table-3).
+
+<a name="table-3"></a>
+
+**Header**   | **Description**
+-------------| ----------------------
+bearer-token | authenticates client at a server
+origin       | indicates origin of request
+cookie       | contains cookies set by server
+
+**Table 3.:** Example non-whitelisted CORS headers.
+
+Attaching non-whitelisted headers to CORS requests is discouraged by the HTML standard and servers assume that cross-origin requests contain only whitelisted headers. Sending non-whitelisted headers from cross-origin domains would allow malicious third-party apps to craft headers that misuse user cookies that Chrome (or another browser) stores and attaches to requests. The cookies could authenticate malicious server transactions that would otherwise not be possible.
+
+### Attaching Headers to Custom Tabs CORS Requests
+[Custom Tabs][8] are a special way of launching web pages in a customised Chrome tab. Custom Tab intents can be created using `CustomTabsIntent.Builder()`. Headers can also be attached to these intents using a Bundle with the [`Borwser.EXTRA_HEADERS flag`][9]:
+
+
+```Java
+CustomTabsIntent intent = new CustomTabsIntent.Builder(session).build();
+
+Bundle headers = new Bundle();
+headers.putString("bearer-token", "Some token");
+headers.putString("redirect-url", "Some redirect url");   intent.intent.putExtra(Browser.EXTRA_HEADERS, headers);
+
+intent.launchUrl(Activity.this, Uri.parse("http://www.google.com"));
+```
+
+Whitelisted headers can always be attached to custom tabs CORS requests. However, Chrome filters non-whitelisted headers by default. 
```

Comment:
What about other browsers? Maybe we could mention that other browsers may have different behaviour but, overall, developers should expect non-whitelisted headers to be blocked.